tox.ini

```diff
@@ -1,14 +1,22 @@
 [tox]
-envlist = py38-django{32}-drf3
-    py39-django{32,41,42}-drf3
-    py310-django{32,41,42}-drf3
-    py311-django{41,42}-drf3
+envlist =
+    py38-django{32}
+    py39-django{32,41,42}
+    py310-django{32,41,42,50}
+    py311-django{41,42,50}
+    py312-django{42,50}
```

Comment:
I dropped the `drf3` from the environment names because there’s no split of versions, and it doesn’t seem like DRF will ever have a version 4 now.